playground/backend/internal/code_processing/code_processing.go

```diff
@@ -109,6 +116,15 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel, nil, &runError, errorChannel, pb.Status_STATUS_RUN_ERROR, pb.Status_STATUS_FINISHED)
 }
 
+func SetupError(err error, pipelineId uuid.UUID, cacheService cache.Cache, ctxWithTimeout context.Context) bool {
+	if err != nil {
+		logger.Errorf("%s: error during setup validate builder: %s\n", pipelineId, err.Error())
```

Comment:
`SetupError()` is used for several cases but the log says that error in the `validate` step